Backpack/src/main/java/net/skyscanner/backpack/text/BpkLinkSpan.kt

```diff
@@ -0,0 +1,25 @@
+package net.skyscanner.backpack.text
+
+import android.content.Context
+import android.text.TextPaint
+import android.text.style.ClickableSpan
+import android.view.View
+import net.skyscanner.backpack.util.BpkTheme
+
+class BpkLinkSpan<T>(
+  context: Context,
+  private val link: T,
+  private val linksHandler: (T) -> Unit
```

Comment:
nit: shouldn't this be called `linkHandler` (singular) as it refers to only one link?

Reply:
Done